### PR TITLE
jpeg-xl 0.8.0

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by chenrui333** [comment]
build failure

```
/tmp/jpeg-xl-20230118-9380-gkbm7a/libjxl-0.8.0/lib/jxl/enc_color_management.cc: In member function ‘jxl::Status jxl::ColorEncoding::SetFieldsFromICC()’:
  /tmp/jpeg-xl-20230118-9380-gkbm7a/libjxl-0.8.0/lib/jxl/enc_color_management.cc:899:15: error: ‘skcms_ICCProfile’ {aka ‘struct skcms_ICCProfile’} has no member named ‘has_CICP’
    899 |   if (profile.has_CICP && ApplyCICP(profile.CICP.color_primaries,
        |               ^~~~~~~~
  /tmp/jpeg-xl-20230118-9380-gkbm7a/libjxl-0.8.0/lib/jxl/enc_color_management.cc:899:45: error: ‘skcms_ICCProfile’ {aka ‘struct skcms_ICCProfile’} has no member named ‘CICP’
    899 |   if (profile.has_CICP && ApplyCICP(profile.CICP.color_primaries,
        |                                             ^~~~
  /tmp/jpeg-xl-20230118-9380-gkbm7a/libjxl-0.8.0/lib/jxl/enc_color_management.cc:900:45: error: ‘skcms_ICCProfile’ {aka ‘struct skcms_ICCProfile’} has no member named ‘CICP’
    900 |                                     profile.CICP.transfer_characteristics,
        |                                             ^~~~
  /tmp/jpeg-xl-20230118-9[380](https://github.com/Homebrew/homebrew-core/actions/runs/3952175273/jobs/6767015774#step:7:381)-gkbm7a/libjxl-0.8.0/lib/jxl/enc_color_management.cc:901:45: error: ‘skcms_ICCProfile’ {aka ‘struct skcms_ICCProfile’} has no member named ‘CICP’
    901 |                                     profile.CICP.matrix_coefficients,
        |                                             ^~~~
  /tmp/jpeg-xl-20230118-9380-gkbm7a/libjxl-0.8.0/lib/jxl/enc_color_management.cc:902:45: error: ‘skcms_ICCProfile’ {aka ‘struct skcms_ICCProfile’} has no member named ‘CICP’
    902 |                                     profile.CICP.video_full_range_flag, this)) {
        |                                             ^~~~
  gmake[2]: *** [lib/CMakeFiles/jxl_enc-obj.dir/build.make:247: lib/CMakeFiles/jxl_enc-obj.dir/jxl/enc_color_management.cc.o] Error 1
  gmake[2]: Leaving directory '/tmp/jpeg-xl-20230118-9380-gkbm7a/libjxl-0.8.0/build'
  gmake[1]: *** [CMakeFiles/Makefile2:499: lib/CMakeFiles/jxl_enc-obj.dir/all] Error 2
  gmake[1]: *** Waiting for unfinished jobs....
```

**Comment by chenrui333** [comment]
upload bottle step failed for `13`

image

**Comment by chenrui333** [comment]
rerun the failed job 🤞

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenrui333** [comment]
just networking failure for `opentsdb` on 11, good to go now.

File: Formula/jpeg-xl.rb [hunk 1]
```diff
@@ -41,26 +43,22 @@ class JpegXl < Formula
...
-        revision: "64374756e03700d649f897dbd98c95e78c30c7da"
+        revision: "b25b07b4b07990811de121c0356155b2ba0f4318"
   end
 
   def install
     resources.each { |r| r.stage buildpath/"third_party"/r.name }
     # disable manpages due to problems with asciidoc 10
     system "cmake", "-S", ".", "-B", "build",
                     "-DJPEGXL_FORCE_SYSTEM_BROTLI=ON",
+                    "-DJPEGXL_FORCE_SYSTEM_LCMS2=ON",
```
Comment (kleisauke) [inline Formula/jpeg-xl.rb:61]: FWIW, libjxl would only use LCMS (`depends_on "little-cms2"`) when building with `-DJPEGXL_ENABLE_SKCMS=OFF`. If that's preferred, you can safely remove:
```ruby
  resource "skcms" do
    url "https://skia.googlesource.com/skcms.git",
        revision: "b25b07b4b07990811de121c0356155b2ba0f4318"
  end
```

Comment (kleisauke) [reply Formula/jpeg-xl.rb:61]: For reference, LCMS is used by Debian, while Fedora prefers skcms.
https://salsa.debian.org/debian-phototools-team/libjxl/-/blob/d5e9a5f19ead9dd70756fc0451a3309eee1cee08/debian/rules#L64
https://src.fedoraproject.org/rpms/jpegxl/blob/3b31ff8895fe7b15a7af6c89162fc96feaa71d48/f/jpegxl.spec#_79

**Pull request closed without merging**